Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -1426,17 +1435,37 @@ - (void)didIgnoreFulfillmentWithNoAuthorizationPresent
 #pragma mark - LCP
 
 /// Fulfill LCP license
-/// @param licenseUrl Downloaded LCP license URL
+/// @param fileUrl Downloaded LCP license URL
 /// @param book `NYPLBook` Book
-- (void)fulfillLCPLicense:(NSURL *)licenseUrl
+/// @param downloadTask download task
+- (void)fulfillLCPLicense:(NSURL *)fileUrl
                   forBook:(NYPLBook *)book
              downloadTask:(NSURLSessionDownloadTask *)downloadTask
 {
   #if defined(LCP)
   LCPLibraryService *lcpService = [[LCPLibraryService alloc] init];
+  // Ensure correct license extension
+  NSURL *licenseUrl = [[fileUrl URLByDeletingPathExtension] URLByAppendingPathExtension:lcpService.licenseExtension];
+  NSError *moveError;
+  [[NSFileManager defaultManager] moveItemAtURL:fileUrl toURL:licenseUrl error:&moveError];
```

Comment:
This is not a book, it's the book's license file, LCP needs `.lcpl` extension for the file. Replaced this code with `replaceItemAtURL...` to avoid possible issues.